docs/style-guide.rst

```diff
@@ -1058,8 +1058,9 @@ Inside each contract, library or interface, use the following order:
 1. Type declarations
 2. State variables
 3. Events
-4. Modifiers
-5. Functions
+4. Errors
```

Comment:
Do we have any examples of errors in the style guide? If not, I would move this change to another issue/PR pair where the `4. Error` entry can be added, with some examples.

Reply:
Added example at f18f4f06d4f94cb680f2ad661f872baa6fc9afd8